app/uk/gov/hmrc/traderservices/models/CaseSLA.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.traderservices.models
+
+import play.api.libs.json.{Format, Json}
+import java.time.LocalDateTime
+
+/**
+  * An expected case response time.
+  *
+  * Route = Hold - do not show an SLA
+  * Route = anything other than Hold  - show SLA
+  *
+  * Air transportation exports = 2 hours
+  * RORO transportation exports = 2 hours
+  * Maritime exports = 2 hours
+  *
+  * Air transportation imports = 2 hours
+  * RORO transportation imports = 2 hours
+  * Maritime transportation imports (received before 15:00) = 3 hours
+  * Maritime transportation imports  (received after 15:00) = 08:00 the next day
+  */
+case class CaseSLA(dateTime: Option[LocalDateTime])
+
+object CaseSLA {
+
+  def calculateFrom(sumbissionDateTime: LocalDateTime, questionsAnswers: QuestionsAnswers): CaseSLA = {
+    val slaDateTime: Option[LocalDateTime] = questionsAnswers match {
+      case answers: ExportQuestions =>
+        answers.routeType.flatMap {
+          case ExportRouteType.Hold => None
+          case _ =>
+            answers.freightType.map {
+              case ExportFreightType.Air      => sumbissionDateTime.plusHours(2)
+              case ExportFreightType.RORO     => sumbissionDateTime.plusHours(2)
+              case ExportFreightType.Maritime => sumbissionDateTime.plusHours(3)
+            }
+        }
+      case answers: ImportQuestions =>
+        answers.routeType.flatMap {
+          case ImportRouteType.Hold => None
+          case _ =>
+            answers.freightType.map {
+              case ImportFreightType.Air  => sumbissionDateTime.plusHours(2)
+              case ImportFreightType.RORO => sumbissionDateTime.plusHours(2)
+              case ImportFreightType.Maritime =>
+                if (sumbissionDateTime.isAfter(sumbissionDateTime.withHour(15).withMinute(0).withSecond(0).withNano(0)))
+                  sumbissionDateTime.plusDays(1).withHour(8).withMinute(0).withSecond(0).withNano(0)
```

Comment:
Works well overall but after midnight it uses the normal logic.

ie. 
Submit at 23.59 => 8.00
Submit at 00.00 => 3.00 

I double checked with Meg and we want to run this up until 5am so it has the standard 3hrs, so if a case goes in at: 
1500 - 0459 => 8.00
0500+ => return to standard logic (+3 hrs)

Reply:
ok, but this is new logic criteria

Reply:
Yeah again not detailed in story but I will add - thank you